SPECS/kernel-uvm/kernel-uvm.spec

```diff
@@ -137,6 +137,9 @@ cp .config %{buildroot}%{_prefix}/src/linux-headers-%{uname_r} # copy .config ma
 ln -sf "%{_prefix}/src/linux-headers-%{uname_r}" "%{buildroot}/lib/modules/%{uname_r}/build"
 find %{buildroot}/lib/modules -name '*.ko' -exec chmod u+x {} +
 
+mkdir -p %{buildroot}/boot/kernel-uvm/
+install -vm 600 arch/%{arch}/boot/bzImage %{buildroot}/boot/kernel-uvm/bzImage
```

Comment:
variable for bzImage? something similar to %{image_fname}?

Reply:
Binary moved out of /boot and into same directory as vmlinux.bin

Reply:
I don't think this filename changes at all, so not sure if a variable is necessary for this one.